index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+	<article class="product">
+	  <section class="s1">
```

Comment:
**Pro tip:** Have descriptive class names. They should reflect the content that will be in the HTML element, not their current sequence.  If the sequence ever changes, it would confusing to other developers.